features/step_definitions/api_steps.rb

```diff
@@ -155,7 +150,7 @@
 end
 
 def lookup_work_package(work_package_name)
-  work_package = decoded_json['planning_elements'].select { |wp| wp['subject'] == work_package_name }.first
+  work_package = decoded_json['planning_elements'].find { |wp| wp['subject'] == work_package_name }
```

Comment:
Useless assignment to variable - `work_package`. Did you mean `work_package_name`?<br>Prefer `detect` over `find`.